### ISSUE TITLE
HTTPS

### ISSUE THREAD

**Issue by Madis0** [jrbudda/Vivecraft_115#3]
Please add HTTPS to your site. See https://letsencrypt.org/ or https://httpsiseasy.com/

**Comment by Techjar** [comment]
Yeah go bitch at GoDaddy to stop charging extra for SSL certificates.

**Comment by Madis0** [comment]
I see. Well, the second link would still work without manual interactions.

**Comment by Techjar** [comment]
Yeah no, with GoDaddy's hosting you have to either buy their certs or upload your own cert manually. The fully manual process for Let's Encrypt certs is a huge pain, and there's no way to automate the renewal so that would have to be repeated every 2-3 months. Not happening. If you want to bring up the point about using a better host, go argue with @jrbudda about that one, but honestly you'll be talking to a wall. I've brought it up many times to no avail.

**Comment by Madis0** [comment]
What I'm saying is that using Cloudflare (or some other CDN) would prevent the cert upload need, as it would be handled by the CDN itself.

**Comment by Techjar** [comment]
I don't think it's possible to use that with GoDaddy hosting.

**Comment by LawnCable** [comment]
why are you using godaddy hosting anyway?
